src/engines/html5/media-source/adapters/native-adapter.js

```diff
@@ -581,11 +604,13 @@ export default class NativeAdapter extends BaseMediaSourceAdapter {
     if (videoTracks) {
       for (let i = 0; i < videoTracks.length; i++) {
         const settings = {
-          //TODO calculate width/height/bandwidth
+          //TODO calculate bandwidth
           id: videoTracks[i].id,
           active: videoTracks[i].selected,
           label: videoTracks[i].label,
           language: videoTracks[i].language,
+          width: videoTracks[i].width | this._videoElement.videoWidth,
+          height: videoTracks[i].height | this._videoElement.videoHeight,
```

Comment:
this will only include, at best, one track so see previous comment - I think we can omit this.

Reply:
Why not? it'll indicate which track is playing

Reply:
see my other comment - the intention of this requirement is to have the transient event logged, and not accumulate them in the tracks object.